t/cobrand/bromley_waste.t

```diff
@@ -378,7 +378,97 @@ FixMyStreet::override_config {
         $mech->content_contains('You have a pending garden subscription');
         $mech->content_lacks('Subscribe to Green Garden Waste');
     };
+};
 
+for my $test (
+{
+    config => { start_date => '2020-01-29'},
+    data => {
+        renewal_text => '£20.00 per bin per year',
+        test_text => 'Renewal price picks up higher cost when renewal date after price rise day',
+    },
+},
+{
+    config => { start_date => '2021-07-14'},
+    data => {
+        renewal_text => '£20.00 per bin per year',
+        test_text => 'Renewal price picks up higher cost when renewal date on price rise day',
+    },
+},
+{
+    config => { start_date => '2021-07-15'},
+    data => {
+        renewal_text => '£10.00 per bin per year',
+        test_text => 'Renewal price picks up lower cost when renewal date before price rise day',
+    },
+}) {
+FixMyStreet::override_config {
+    ALLOWED_COBRANDS => 'bromley',
+    COBRAND_FEATURES => {
+        payment_gateway => { bromley => { ggw_cost => [
+            { start_date => '2019-02-27', cost => 1000 },
+            { start_date => $test->{config}->{start_date}, cost => 2000 },
+            ]}},
+        echo => { bromley => { sample_data => 1 } },
+        waste => { bromley => 1 }
+    },
+}, sub {
+    subtest $test->{data}{test_text} => sub {
+        my $echo = Test::MockModule->new('Integrations::Echo');
+        $echo->mock('GetServiceUnitsForObject', sub {
```

Comment:
This mock and the set_fixed_time are the same for every iteration, so can be done once before the whole $test loop, I think?